scss/reset/_headings.scss

```diff
@@ -66,13 +64,29 @@ the tag as a heading.
 .text--display2 {
 	@extend %text--display2;
 }
-.text--display2 {
```

Comment:
this is a breaking change; we'll need to leave the class in until we clean it up in consumer apps